includes/services/Leaflet/jquery.leaflet.js

```diff
@@ -25,7 +25,9 @@
 		* @param {Object} markerData Contains the fields lat, lon, title, text and icon
 		* @return {L.Marker}
 		*/
-		this.addMarker = function (properties) {
+		this.addMarker = function (properties, dontAddToMap) {
```

Comment:
See this https://github.com/JeroenDeDauw/Maps/pull/194/files#diff-4a1096507c3fea5c0fb17f1231c4a617R179 vs. this
https://github.com/JeroenDeDauw/Maps/pull/194/files#diff-4a1096507c3fea5c0fb17f1231c4a617R184 .

For the cluster the markers are not directly added to the map but to the cluster thing. I could also rename it to `addToMap` if you want. ;)